### ISSUE TITLE
wrong error for keyspacing

### ISSUE THREAD

**Issue by codvio** [eslint/eslint#5263]
Test case:

```javascript
    var params = {
      method : 'post',
      url    : '/test',
      timeout: 15000,
      headers: {
        Authorization: 'xyz'
      },
      data   : {}
    };
```

error:
```
Extra space after key "data"    key-spacing
```

**Comment by eslintbot** [comment]
Hi @codvio, thanks for the issue. It looks like there's not enough information for us to know how to help you. If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

If it's something else, please just provide as much additional information as possible. Thanks!

**Comment by btmills** [comment]
This is a duplicate of #2668, which is accepted and just needs someone to implement it. @codvio want to send a PR?
